### PR TITLE
Newiki

### PR DESCRIPTION
**Pull request by Herrgrim0**
goodbye django-wiki, hello django-reversion !

### CODE REVIEW

File: incubator/settings.py [hunk 1]
```diff
@@ -216,7 +215,7 @@
 
 LOGIN_URL = '/auth/login/'
 
-WIKI_ATTACHMENTS_EXTENSIONS = (
+"""WIKI_ATTACHMENTS_EXTENSIONS = (
```
Comment (C4ptainCrunch) [inline incubator/settings.py]: Don't comment, just remove the lines, if you need it back it's still in git ;)

File: requirements.txt [hunk 2]
```diff
@@ -23,5 +23,7 @@ django-jsonfield-compat
 pandas
 django-pandas
 pip>8.0
+django-reversion<3.0
```
Comment (etnarek) [inline requirements.txt]: Do we still need that one?

Comment (Herrgrim0) [reply requirements.txt]: nope :)

**Comment by Mortinat** [comment]
Implémenter durant l'upgrade vers django3

**Pull request closed without merging**